ui/css/utilities/colors.scss

```diff
@@ -8,6 +8,8 @@
   --goerli: #3099f2;
   --localhost: #bbc0c5;
   --flask-purple: #8b45b6;
+  --inherit: inherit;
+  --transparent: transparent;
 
   // DO NOT CHANGE
   // Required for the QR reader to work properly
```

Comment:
This should be taken care of in https://github.com/MetaMask/metamask-extension/pull/15507

Reply:
cc @NidhiKJha